src/main/java/ru/yandex/practicum/filmorate/controller/GenreController.java

```diff
@@ -0,0 +1,30 @@
+package ru.yandex.practicum.filmorate.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ru.yandex.practicum.filmorate.model.Genre;
+import ru.yandex.practicum.filmorate.service.GenreService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/genres")
+@AllArgsConstructor
+public class GenreController {
+
+    private GenreService genreService;
+
+    @GetMapping
+    private List<Genre> getAllGenres() {
```

Comment:
Они private и в других классах (UserController, FilmController). В данный момент не вижу смысла давать 
к ним доступ. 
Заменил все модификаторы на public. 
Нашёл только вот такую цитату в пользу использования public вот [здесь](https://www.baeldung.com/java-access-modifiers#:~:text=We%20should%20always%20use%20the,is%20part%20of%20an%20API.) :

> It’s good practice to use the most restrictive access level possible for any given member to prevent misuse. We should always use the private access modifier unless there is a good reason not to. **Public access level should only be used if a member is part of an API**.

На реальных проектах обычно ставят public?

Reply:
Роль контроллера как раз и заключается в том, чтобы общаться с внешней средой, а значит иметь модификатор private
Если же такой необходимости нет (иметь сообщение во вне), то и контроллер не нужен, а нужен сервис, который можно инкапсулировать

Reply:
Оки, принято